frontend/scripts/index.ejs

```diff
@@ -6,6 +6,7 @@
 <%= htmlWebpackPlugin.options.icons %>
 
 <div id="app-root-container"></div>
+<div id="app-portal-container"></div>
```

Comment:
Don't really see the point of using a portal here. Are you sure this is the best option?